proj/silencer.conf

```diff
@@ -1,10 +1,11 @@
 // https://github.com/ghik/silencer.git#787144f5557421017263c1ea36c22f4c078d24cb  # was master
```

Comment:
you'll need to edit this line as well, in order for `advance` not to wipe out the change you made below